crypter.go

```diff
@@ -28,6 +28,7 @@ import (
 type Encrypter interface {
 	Encrypt(plaintext []byte) (*JSONWebEncryption, error)
 	EncryptWithAuthData(plaintext []byte, aad []byte) (*JSONWebEncryption, error)
+	Options() *EncrypterOptions
```

Comment:
Why return a pointer here? It looks like the resulting options struct is read-only.